source/install/install-ubuntu-1404-mattermost.rst

```diff
@@ -0,0 +1,95 @@
+.. _install-ubuntu-1404-mattermost:
+
+Installing Mattermost Server
+============================
+
+Install Mattermost Server on a 64-bit machine.
+
+Assume that the IP address of this server is 10.10.10.2
+
+**To install Mattermost Server on Ubuntu**
+
+1. Log into the server that will host Mattermost Server and open a terminal window.
+
+2. Create a mattermost user and group
+
+    - ``sudo adduser --system --group mattermost``
+
+3. Change to the mattermost home directory.
+
+    ``cd /home/mattermost``
+
+4. Download `the latest version of the Mattermost Server <https://docs.mattermost.com/administration/upgrade.html#version-archive>`_ by typing:
+
+   -  ``wget https://releases.mattermost.com/X.X.X/mattermost-X.X.X-linux-amd64.tar.gz``
+   -  Where ``vX.X.X`` is the latest version.
+
+5. Extract the Mattermost Server files by typing:
+
+   -  ``sudo tar -xvzf *.gz``
+
+6. Change the user and group of the extracted files to mattermost
+
+   - ``sudo chown -R mattermost:mattermost mattermost/``
+
+7. Create the storage directory for files. We assume you will have
+   attached a large drive for storage of images and files. For this
+   setup we will assume the directory is located at
+   ``/mattermost/data``.
+
+   -  Create the directory by typing:
+   -  ``sudo mkdir -p /mattermost/data``
+   -  Set the mattermost account as the directory owner by typing:
+   -  ``sudo chown -R mattermost:mattermost /mattermost``
+
+8. Configure Mattermost Server by editing the config.json file at
+   ``/home/mattermost/config``
+
+   -  ``cd /home/mattermost/mattermost/config``
+   -  Edit the file by typing:
+   -  ``vi config.json``
+   -  If you are using PostgreSQL:    
+     -  Set ``DriverName":`` to ``"postgres"``
+     -  Set ``"DataSource:"`` to the following value: ``"postgres://mmuser:mmuser_password@10.10.10.1:5432/mattermost?sslmode=disable&connect_timeout=10"``
+   -  If you are using MySQL:    
+     -  Set ``DriverName":`` to ``"mysql"``
+     -  Set ``"DataSource":`` to the following value: ``"mmuser:mmuser_password@tcp(10.10.10.1:3306)/mattermost?charset=utf8"``
+   -  You can continue to edit configuration settings in
+      ``config.json`` or use the System Console described in a later
+      section to finish the configuration.
+
+9. Test the Mattermost Server
+
+   -  ``cd ~/mattermost/bin``
+   -  Run the Mattermost Server by typing:
+   -  ``./platform``
```

Comment:
Thanks! I updated the command.

I made the Mattermost server config as the next section because at this point, the user has just finished installing the server, so in terms of flow, it's better to go straight to the config, then move on to installing other components.